### PR TITLE
Add a pull request template

### PR DESCRIPTION
**Pull request by stormi**
The template asks users to check two boxes, as a way to make them aware
of the DCO and under which license they put their contribution.

To be discussed.

### CODE REVIEW

**Comment by stormi** [comment]
Not "WIP" anymore.

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+
+> Before submitting the pull request, you must agree with the following statements by checking the boxes with a 'x'.
```
Comment (olivierlambert) [inline pull_request_template.md:3]: "by checking **all** the boxes" maybe? (to be sure both need to be checked)

Comment (stormi) [reply pull_request_template.md]: Done, replaced "the boxes" with "both boxes".

**Comment by olivierlambert** [comment]
LGTM!

**Pull request merged**